### PR TITLE
New downloads page

### PR DESCRIPTION
**Pull request by SergioRAgostinho**
See it live here http://sergioagostinho.com/PointCloudLibrary.github.io/downloads/

My desktop was caching the page so be sure to refresh properly.

If you need to apply corrections, feel free to commit directly to my branch.

### CODE REVIEW

**Comment by SergioRAgostinho** [comment]
Needs proper proof reading. Some prepositions are off.

File: downloads.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+PS> .\vcpkg install pcl
+```
+
+This is our recommended installation method for **Windows** users.
+
+### Homebrew
+
+[![Homebrew package](https://repology.org/badge/version-for-repo/homebrew/pcl-pointclouds.svg)](https://formulae.brew.sh/formula/pcl)
+
+[Homebrew](https://brew.sh/) became known as being missing package manager for macOS, but in recent years it extended its support to Linux. To install PCL on Homebrew-enabled desktops type the following on your terminal
```
Comment (kunaltyagi) [inline downloads.md:46]:
```suggestion
[Homebrew](https://brew.sh/) became known as being missing package manager for macOS, but in recent years it has extended its support to Linux. To install PCL on Homebrew-enabled desktops, type the following in your terminal
```

File: downloads.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+## Cross-Platform
+
+In case your OS does not have a dedicated package manager, consider one of the following options.
+
+### vcpkg
+
+[![Vcpkg package](https://repology.org/badge/version-for-repo/vcpkg/pcl-pointclouds.svg)](https://github.com/Microsoft/vcpkg/tree/master/ports/pcl)
+
+[vcpkg](https://github.com/microsoft/vcpkg) is a cross-platform open source package manager created by Microsoft, available for Windows, Linux and macOS. To install PCL on vcpkg-enabled desktops type the following on your terminal
```
Comment (kunaltyagi) [inline downloads.md:34]:
```suggestion
[vcpkg](https://github.com/microsoft/vcpkg) is a cross-platform open source package manager created by Microsoft, available for Windows, Linux and macOS. To install PCL on vcpkg-enabled desktops, type the following in your terminal
```

File: downloads.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+---
+layout: page
+title: Downloads
+permalink: /downloads/
+---
+
+## Prebuilt binaries
+
+
+Point Cloud Library (PCL) runs on many operating systems, and prebuilt binaries are available for Linux, Windows, and macOS. In addition to installing PCL, you will need to download and compile a set of 3rd party libraries that PCL requires in order to function. To sidestep all that trouble, we recommend you to install PCL through one of the many available package managers out there. Package managers are the preferred distribution mechanism to download PCL. We provide a couple of options below dependent on your platform of choice.
```
Comment (kunaltyagi) [inline downloads.md:10]:
```suggestion
Point Cloud Library (PCL) runs on many operating systems, and prebuilt binaries are available for Linux, Windows, and macOS. You also need to setup and provide a set of 3rd party libraries required by PCL.
To sidestep all that trouble, we recommend you to install PCL through one of the many available package managers out there. Package managers are the preferred distribution mechanism to download PCL.
We provide a number of options based on your platform of choice.
```

**Comment by kunaltyagi** [comment]
Closes #4

**Pull request merged**